radio/src/gui/480x272/model_setup.cpp

```diff
@@ -1013,13 +1026,15 @@ bool menuModelSetup(event_t event)
         }
       }
       break;
-    case  ITEM_MODEL_EXTERNAL_MODULE_POWER: {
+
+    case ITEM_MODEL_EXTERNAL_MODULE_POWER:
+    {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Extract this nested code block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS1199)